src/main/java/gtPlusPlus/xmod/gregtech/loaders/recipe/RecipeLoader_ChemicalSkips.java

```diff
@@ -260,10 +276,26 @@ private static void quantumTransformerRecipes() {
                         Materials.Grade6PurifiedWater.getFluid(10000L), Materials.Grade7PurifiedWater.getFluid(5000L),
                         Materials.Grade8PurifiedWater.getFluid(1000L) },
                 new ItemStack[] {},
-                new int[] { 2000, 2000, 2000, 2000, 2000 },
+                new int[] { 2500, 2500, 2500, 2500 },
                 20 * 20,
                 (int) TierEU.RECIPE_UIV,
                 4);
+
+        // Rare Particles
+        CORE.RA.addQuantumTransformerRecipe(
+                new ItemStack[] { ItemUtils.getSimpleStack(GenericChem.mSynchrotronCapableCatalyst, 0),
+                        GregtechItemList.Laser_Lens_Special.get(1) },
+                new FluidStack[] { FluidUtils.getFluidStack("plasma.hydrogen", 30000),
+                        Materials.Helium.getPlasma(30000L), Materials.Beryllium.getPlasma(30000L),
```

Comment:
please change the beryllium plasma here too

Reply:
Good catch, I forgot I had added the same plasma to the QFT recipe. Fixed.